src/forms/FormFieldImageVideoLinkDocumentBrowsing/FormFieldImageVideoLinkDocumentBrowsing.tsx

```diff
@@ -160,39 +160,45 @@ const FormFieldImageVideoLinkDocumentBrowsing = (
 		<ImageVideoLinkDocumentBrowsingContainer>
 			{(Array.isArray(value) && value?.length === 0) || !value ? (
 				<BrowsingContainer>
-					<BrowseSpan>
+					<BrowseSpan
+						className = { hasOptions ? "has-options" : "" }

```

Comment:
I don't think BrowseSpan should change it's style depending on whether there are options or not. The one that should be changing is BrowseOptionsContainer since it's optional.